src/components/Loader/PeopleTable/PersonItem.tsx

```diff
@@ -0,0 +1,55 @@
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../../types';
+import classNames from 'classnames';
+
+export const PersonItem = ({ person }: { person: Person }) => {
```

Comment:
It would be better to create a type or interface instead of trying to type your person right here.